packages/aws-cdk/lib/api/deploy-stack.ts

```diff
@@ -416,6 +424,7 @@ class FullCloudFormationDeployment {
       ResourcesToImport: this.options.resourcesToImport,
       Description: `CDK Changeset for execution ${this.uuid}`,
       ClientToken: `create${this.uuid}`,
+      ImportExistingResources: importExistingResources,
```

Comment:
what happens if they specify both `ResourcesToImport` and `ImportExistingResources`?

Reply:
I guess it will not work, but [the spec](https://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_CreateChangeSet.html) says nothing about such restriction, so I would like to keep it as-is. 

Actually users cannot set both ImportExistingResources and ResourcesToImport from the CDK CLI, because the former is only used in cdk deploy, and cdk deploy does not have any command argument to directly set ResourcesToImport.

Reply:
The spec isn't really the source of truth we all wish it was so it's not uncommon we enforce logical things the spec does not. However in this specific instance I believe this is correct because there is a difference between the resources supported by `importExistingResources` and `ResourcesToImport`. Auto-import is a subset of resources supported by `ResourcesToImport` so it is entirely possible to have both. I believe this is fine to leave as is